velox/docs/configs.rst

```diff
@@ -43,6 +43,16 @@ Generic Configuration
      - integer
      - 80
      - Abandons partial TopNRowNumber if number of output rows equals or exceeds this percentage of the number of input rows.
+   * - abandon_build_no_dup_hash_min_rows
+     - integer
+     - 100,000
+     - Number of input rows to receive before starting to check whether to abandon building a HashTable without
+       duplicates in HashBuild for left semi/anti join.
+   * - abandon_build_no_dup_hash_min_pct
```

Comment:
Just a nit: the current config name is a bit confusing. How about 'dedup hashmap' to better reflect the underlying change for this feature?

Reply:
Yes, I will change it to `abandon_dedup_hashmap_min_pct`